yarn.lock

```diff
@@ -2,6 +2,110 @@
 # yarn lockfile v1
 
 
+"@algolia/cache-browser-local-storage@4.5.1":
```

Comment:
these seem to be added by shipjs not updating all versions (of dependencies), let's see what's different than in the script before

it's causing the test:build to fail 👍